### PR TITLE
fix(platform): update settings generator docs and component to update theme/language automatically, add readonly property to form generator

### PR DESCRIPTION
**Pull request by mikerodonnell89**
This is part of #12641, and not intended to be an entire update to the new design. We'll do this incrementally

### CODE REVIEW

**Comment by droshev** [comment]
@dpavlenishvili @khotcholava Can you review the PR?

File: libs/core/input-group/input-group.component.html [hunk 1]
```diff
@@ -81,5 +81,6 @@
         [attr.placeholder]="_cvaControl.cvaDirective?.placeholder || null"
         [attr.aria-required]="required"
         (search)="_onSearchEvent($event)"
+        autocomplete="off"
```
Comment (khotcholava) [inline libs/core/input-group/input-group.component.html:84]: Is it necessary for the input group component to always have autocomplete set to "off"? @mikerodonnell89

**Review by khotcholava — CHANGES_REQUESTED** [review]

File: libs/core/input-group/input-group.component.html (hunk 1 above)
Comment (mikerodonnell89) [reply libs/core/input-group/input-group.component.html:84]: You're right, probably best to just do this specifically for combobox/multi-input etc

File: libs/platform/form/form-generator/base-dynamic-form-generator-control.ts [hunk 2]
```diff
@@ -65,4 +65,13 @@ export abstract class BaseDynamicFormGeneratorControl<T extends BaseDynamicFormF
         }
         return (this.formItem?.useMessageAsPlaceholder && this.formItem?.message) || '';
     }
+
+    /** @hidden */
+    get readonly(): boolean {
```
Comment (dpavlenishvili) [inline libs/platform/form/form-generator/base-dynamic-form-generator-control.ts:70]: you can simplify it:

`
get readonly(): boolean {
    return this.formItem?.readonly ?? false;
}
`

File: libs/platform/settings-generator/layouts/settings-generator-sidebar-layout/settings-generator-sidebar-layout.component.html [hunk 3]
```diff
@@ -5,50 +5,72 @@
...
-                                    width="40%"
-                                    textLines="1"
-                                    [style.margin]="'auto 0'"
-                                ></fd-skeleton>
-                            </div>
+    <div class="fdp-settings-generator__search-header">
+        <h3 fd-title class="fdp-settings-generator__title">
+            {{ 'platformSettingsGenerator.settingsLabel' | fdTranslate }}
+        </h3>
+        <!-- TODO: implement search input
```
Comment (khotcholava) [inline libs/platform/settings-generator/layouts/settings-generator-sidebar-layout/settings-generator-sidebar-layout.component.html:12]: Do we need commented code here? @mikerodonnell89

**Review by droshev — APPROVED** [review]

**Pull request merged**